src/handler.ts

```diff
@@ -3,6 +3,14 @@ import { datadog, datadogHandlerEnvVar, lambdaTaskRootEnvVar, getEnvValue } from
 // tslint:disable-next-line:no-var-requires
 const { load } = require("/var/runtime/UserFunction") as any;
 
+if (getEnvValue("DD_FS_INTEGRATION_ENABLED", "false").toLowerCase() === "false") {
+  if (process.env.DD_TRACE_DISABLED_PLUGINS) {
+    process.env.DD_TRACE_DISABLED_PLUGINS = `${process.env.DD_TRACE_DISABLED_PLUGINS},fs`;
```

Comment:
It seems that this could result in `fs` being included in the list twice. I.e. if a customer adds it to the disabled list, but also does not set `DD_FS_INTEGRATION_ENABLED`.

It also might be confusing if a customer adds `fs` to dd_trace_disabled_plugins, but then also adds `DD_FS_INTEGRATION_ENABLED`. The result would be that it would not be enabled, even though DD_FS_INTEGRATION_ENABLED is true; the two env vars conflict. That might be pretty confusing.

Maybe a simpler approach would be to set the default value of DD_TRACE_ENABLED_PLUGINS to `fs`? And allow the customer to pass in a different list if desired?

Reply:
Sorry, what I said doesn't quite make sense, let me rethink.

Reply:
Great point. Going to make the change to default the env var if none is set.

Reply:
It does make sense. What we can do here is just default the env var to `fs` if none is set.

Reply:
The main thing is that if a customer passes in a list of disabled plugins that includes `fs`, `fs` should be disabled. And conversely, if they pass in a list of disabled plugins that does not include `fs`, `fs` should not be disabled.